Makefile

```diff
@@ -120,3 +120,17 @@ help: ## Show this help.
 		if (/^[a-zA-Z_-]+:.*?##.*$$/) {printf "    ${YELLOW}%-20s${GREEN}%s${RESET}\n", $$1, $$2} \
 		else if (/^## .*$$/) {printf "  ${CYAN}%s${RESET}\n", substr($$1,4)} \
 		}' $(MAKEFILE_LIST)
+
+## Generate:
+generate-v3-models: ## Generate V3 models using go-swagger
```

Comment:
nit: may be specifying explicitely which swagger.json is used will be helpful for review in general